third-party/CMakeLists.txt

```
@@ -108,9 +180,65 @@ function(extractLocalArchive identifier anchor_file archive_path working_directo
   add_custom_target("${identifier}_extractor" DEPENDS "${absolute_anchor_path}" ${additional_anchor_file_paths})
 endfunction()
 
+function(importThirdPartyBinaryLibrary name anchor_file_name)
+  if(DEFINED PLATFORM_LINUX)
+    set(platform_name "linux")
+  elseif(DEFINED PLATFORM_MACOS)
+    set(platform_name "macos")
+  elseif(DEFINED PLATFORM_WINDOWS)
+    set(platform_name "windows")
+  else()
+    message(FATAL_ERROR "Unrecognized system")
+    return()
+  endif()
+
+  set(identifier "thirdparty_${name}")
+  set(base_path "${THIRD_PARTY_PREBUILT_PATH}/${platform_name}-x86_64")
+
+  # Just the name of the file
+  set(relative_anchor_path "${anchor_file_name}")
+
+  if(${ARGC} GREATER 2)
+    foreach(additional_anchor ${ARGN})
+      list(APPEND additional_anchor_rel_paths "${additional_anchor}")
+    endforeach()
+  endif()
+
+  if(additional_anchor_rel_paths)
+    add_osquery_library("${identifier}" INTERFACE IMPORTED GLOBAL)
+    set_target_properties("${identifier}" PROPERTIES INTERFACE_BINARY_DIR "${base_path}")
+
+    list(APPEND libraries "${relative_anchor_path}")
+    list(APPEND libraries "${additional_anchor_rel_paths}")
+
+    foreach(library ${libraries})
+      get_filename_component(library_name "${library}" NAME_WE)
+      add_osquery_library("${identifier}_${library_name}" STATIC IMPORTED GLOBAL)
+      set_target_properties("${identifier}_${library_name}" PROPERTIES IMPORTED_LOCATION "${base_path}/${library}")
+      target_link_libraries("${identifier}" INTERFACE "${identifier}_${library_name}")
+    endforeach()
+  else()
+    add_osquery_library("${identifier}" STATIC IMPORTED GLOBAL)
+    set_target_properties("${identifier}" PROPERTIES IMPORTED_LOCATION "${base_path}/${relative_anchor_path}")
+  endif()
+
+  execute_process(COMMAND "${CMAKE_COMMAND}" -E make_directory "${base_folder}/include")
```

```suggestion
  execute_process(COMMAND "${CMAKE_COMMAND}" -E make_directory "${base_path}/include")
```
Otherwise it doesn't configure